sensor_msgs/Temperature b0
sensor_msgs/Temperature n0
sensor_msgs/Temperature b1
sensor_msgs/Temperature n1
sensor_msgs/Temperature b2
sensor_msgs/Temperature n2
